example/test/BUILD.bazel

```diff
@@ -78,11 +90,21 @@ ruff_test(
     srcs = [":generated_py"],
 )
 
+ruff_test(
+    name = "ruff_should_ignore_generated_filegroup",
+    srcs = [":generated_py_filegroup"],
+)
+
 ruff_test(
     name = "ruff_should_ignore_excluded",
     srcs = ["excluded_py"],
 )
 
+ruff_test(
```

Comment:
it feels like a lot more lines of tests are added here than are warranted to illustrate and verify the small code change, could you distill it down a bit? (I think a single filegroup is sufficient to get the needed "test coverage")